packages/core/src/components/Accordion/Item.tsx

```diff
@@ -13,6 +13,8 @@ export type Props = {
   id: string;
   /** Index amongst a collection of accordion items. */
   index?: number;
+  /** Removes horizontal the item and top padding from item body. */
```

Comment:
lol, i'm sorry. when i chopped up the original sentence, i missed some words.

```
/** Removes horizontal padding from the item and top padding from item body. */
```

Reply:
Haha I didn't proofread, updated!